packages/babel-helper-module-transforms/src/rewrite-live-references.js

```diff
@@ -29,13 +30,24 @@ export default function rewriteLiveReferences(
   }
 
   for (const [local, data] of metadata.local) {
+    const names = data.bindings.map(({ name }) => name);
```

Comment:
Could we 
Could we avoid all of this if we adjusted `rewriteBindingInitVisitor` instead? e.g.

```
export default foo;
```

to
```
var _tmp = foo;
export { _tmp as default };
```

then in `rewriteBindingInitVisitor` we could say if it's a var that is never reassigned, instead of
```
var _tmp = foo;
exports.default = _tmp;
```
do

```
exports.default = foo;
```